test/algorithms/set_operations/intersection/intersection.cpp

```diff
@@ -303,23 +303,8 @@ void test_areal()
 
     TEST_INTERSECTION(issue_893, 1, -1, 473001.5082956461);
 
-    TEST_INTERSECTION(issue_1226, 1, -1, 0.00036722862);
     TEST_INTERSECTION(issue_1229, 0, -1, 0);
```

Comment:
This one `issue_1229` is kept because it fails for difference (wrong result), this was not tested before.
The new test can ignore validity, but not (yet) wrong results.